executor/common_kvm.h

```diff
@@ -5,6 +5,12 @@
 
 // Common KVM-related definitions.
 
+#define INVALID_GPA ((uintptr_t)-1)
+#define ARCH_EXECUTOR_FN_GPA(f, exec_offset)                                                            \
```

Comment:
This will work for data as well, right? It also took me a bit to decipher "GPA" (still not sure if I got it right).
Perhaps a name that includes something with "GUEST_ADDRESS" will be better for future readers.